docs/docs/tutorial/part-5/index.mdx

```diff
@@ -0,0 +1,530 @@
+---
+title: "Part 5: Transform data to use MDX"
+---
+
+import { Announcement, Notification } from 'gatsby-interface'
+import Collapsible from '@components/collapsible'
+import { MdInfo } from 'react-icons/md'
+
+<Notification
+  Icon={MdInfo}
+  tone="WARNING"
+  variant="SECONDARY"
+  content={
+    <div>
+      <p>
+        <strong>Note:</strong> We{"'"}re still working on updating this Tutorial to use Gatsby v3.
+        The first few parts of the new Tutorial have been released here, and subsequent parts
+        will be added as we finish them.
+      </p>
+      <p style={{ marginBottom: 0 }}>
+        If you{"'"}re looking for a full experience in the meantime, you can check out the (slightly outdated){" "}
+        <a href="https://v2.gatsbyjs.com/docs/tutorial/">
+          <strong>Gatsby v2 Tutorial</strong>
+        </a>
+        .
+      </p>
+    </div>
+  }
+/>
+
+## Introduction
```

Comment:
Just thinking out loud here as I'm not sure myself how to best handle this:

It would probably make sense to mention that _generally speaking_ transformer plugins have the nomenclature of `gatsby-transformer-*`. But then couple of lines down below we say that `gatsby-plugin-mdx` is also one 😄 Which is right but for reasons that I can't recall it was named this way.

But it shouldn't confuse people overall.

Reply:
Good idea! Added a note to the beginning.